templates/internal/appName/rpc/rpc.go.tpl

```diff
@@ -109,7 +119,7 @@ func (s *GRPCService) Close(ctx context.Context) error {
 }
 
 // StartServer starts a RPC server with the provided implementation.
-func StartServer(ctx context.Context, service api.Service, opts... grpcx.ServerOption) (*grpc.Server, error) {
+func StartServers(ctx context.Context, servers *Servers, opts... grpcx.ServerOption) (*grpc.Server, error) {
```

Comment:
This will be breaking because you're removing `StartServer`. Can we add back `StartServer` that just calls `StartServers` in a backwards compatible way and add a deprecation warning to the function?

Reply:
Good point. I will add it.